src/exo/LoopIR_scheduling.py

```diff
@@ -303,6 +321,28 @@ def move_back(c):
         return c.prev()
 
 
+# --------------------------------------------------------------------------- #
+# --------------------------------------------------------------------------- #
+# IR Building Helpers
+
+
+def divide_expr(e, quot):
+    assert isinstance(e, LoopIR.expr)
+    if isinstance(quot, int):
+        quot_int = quot
+        quot_ir = LoopIR.Const(quot, e.type, e.srcinfo)
+    elif isinstance(quot, LoopIR.expr):
```

Comment:
This should be `elif isinstance(quot, LoopIR.Const):`

Reply:
Good catch!